components/tools/OmeroWeb/test/integration/test_tags.py

```diff
@@ -167,7 +167,7 @@ def test_add_edit_and_remove_tag(self):
         _post_response(self.django_client, request_url, data)
         _csrf_post_response(self.django_client, request_url, data)
         # Check that tag is removed - short delay to allow async delete
-        sleep(0.1)
+        sleep(1)
```

Comment:
Probably 0.5 seconds would be sufficient.
Elsewhere I do retry, E.g. https://github.com/will-moore/openmicroscopy/blob/2e750cb9a64646e09d7b734059156a6bdd83e9a3/components/tools/OmeroWeb/test/integration/test_tags.py#L236 but still with a sleep. If we retry, how long should we sleep between retries? (and how many retries)?